config/data/paragonUtils.js

```diff
@@ -0,0 +1,160 @@
+const path = require('path');
+const fs = require('fs');
+
+/**
+ * Attempts to extract the Paragon version from the `node_modules` of
+ * the consuming application.
+ *
+ * @param {string} dir Path to directory containing `node_modules`.
+ * @returns {string} Paragon dependency version of the consuming application
+ */
+function getParagonVersion(dir, { isBrandOverride = false } = {}) {
+  const npmPackageName = isBrandOverride ? '@openedx/brand' : '@openedx/paragon';
```

Comment:
In testing this further with the `@edx/brand` alias installed in this repo's `example` MFE app, and exposing the injected `PARAGON_THEME` global variable  within `App.jsx`, the installed brand package is not installed as this function references `@openedx/brand` rather than the aliased `@edx/brand` currently used by MFEs (e.g., [frontend-app-learning](https://github.com/openedx/frontend-app-learning/blob/3a542766d793168751a6bf48b23a4340f1ba7b2a/package.json#L33)).

I might recommend supporting the following to remain both forward-looking for the `@openedx` migration and the currently used `@edx/brand`:

* `@openedx/paragon`
* `@openedx/brand` (brand override)
* `@edx/brand` (brand override)